metsim/metsim.py

```diff
@@ -511,87 +567,85 @@ def _validate_setup(self):
         if len(errs) > 1:
             raise Exception("\n  ".join(errs))
 
-    def write(self, suffix=""):
-        """
-        Dispatch to the right function based on the configuration given
-        """
-        dispatch = {
-            'netcdf': self.write_netcdf,
-            'ascii': self.write_ascii,
-            'data': self.write_data
-        }
-        dispatch[self.params.get('out_fmt', 'netcdf').lower()](suffix)
-
-    def get_nc_output_suffix(self):
-        s, e = self.output.indexes['time'][[0, -1]]
+    # def write(self, suffix=""):
+    #     """
+    #     Dispatch to the right function based on the configuration given
+    #     """
+    #     dispatch = {
+    #         'netcdf': self.write_netcdf,
+    #         'ascii': self.write_ascii,
+    #         'data': self.write_data
+    #     }
+    #     dispatch[self.params.get('out_fmt', 'netcdf').lower()](suffix)
```

Comment:
thoughts on what do with the existing writers? If it was up to me, we'd only support netcdf output.